compiler/hash-ast/src/ast.rs

```diff
@@ -145,12 +145,61 @@ impl<T> Deref for AstNodeRef<'_, T> {
     }
 }
 
-// @@TODO: Ideally we want some useful methods on this and potentially a span that represents
-//         all the nodes inside it. Why is this useful? Well because sometimes we want to represent
-//         a collection ob sub nodes that might have a larger parent span like for example arguments
-//         in a function def. The entire AstNodes might include the span of the parenthesees but each
-//         argument might just have the span of the argument name and type.
-pub type AstNodes<'c, T> = Row<'c, AstNode<'c, T>>;
```

Comment:
I would prefer to keep this comment or make a gh issue about parent span

Reply:
See #107.